### DISCUSSION TITLE
Ambiant dialogue based on HP%

### DISCUSSION THREAD

**Discussion by Algarad** [Albeoris/Memoria#544] [Q&A]
Hi,

I am trying to create a condition that triggers when a character receives heals when under a specific health percentage, let's say 50%.

I am using:
CheckAnyStatus(CurrentStatus, BattleStatus_LowHP)
but this triggers when health goes yellow (15%?).

It should be possible wiwth NCalc flag TargetHP, but I cannot manage it to work. Any suggestions?

Thanks!

**Comment by SamsamTS** [comment]
BattleStatus_LowHP only triggers at 15% HP. This cannot be changed as far as I know.

However it's possible to detect when HP goes below 25% with this
```
>Hitted PLAYERNAME
VoicePath:FILE
[Condition] (EffectTargetFlags & CalcFlag_HpDamageOrHeal) == 1 && (1.0 * (HP + HPDamage) /MaxHP) > 0.25 && (1.0 * HP / MaxHP) <= 0.25 [/Condition]
```
- `(EffectTargetFlags & CalcFlag_HpDamageOrHeal) == 1` checks if it's damage (not heal)
- `(1.0 * (HP + HPDamage) /MaxHP) > 0.25` checks if HP was above 25% (the multiplication by 1.0 is for converting to float)
- `(1.0 * HP / MaxHP) <= 0.25` checks if HP is below 25%

**Comment by Algarad** [comment]
Hi SamsamTS,
It works like a charm, thank you!
